app/controllers/api/v1/comments_controller.rb

```diff
@@ -14,6 +14,11 @@ def create
       render json: @comment
     end
 
+    def destroy
```

Comment:
Something we haven't been doing so far is making sure only admins can do certain things at the API level, too. Might be good to do that here. You could add a `return head(:forbidden) unless current_user.is_admin` at the top of this function, which will cause this to return a 403 if they're not an admin.